pkg/controllers/localbuild/gitea.go

```diff
@@ -212,7 +215,7 @@ func getGiteaToken(ctx context.Context, baseUrl, username, password string) (str
 	return token.Token, nil
 }
 
-func giteaBaseUrl(config v1alpha1.BuildCustomizationSpec) string {
+func (r *LocalbuildReconciler) GiteaBaseUrl(config v1alpha1.BuildCustomizationSpec) string {
```

Comment:
This doesn't need to be a method and doesn't need to be exported.

Reply:
See: 3ab4fd3